packages/components/src/font-size-picker/index.js

```diff
@@ -47,23 +47,41 @@ function FontSizePicker(
 		availableUnits: [ 'px', 'em', 'rem' ],
 	} );
 
-	// The main font size UI displays a toggle group when the presets are less
-	// than six and a select control when they are more.
-	//
-	// A select control is also used when the value of a preset cannot be
-	// immediately computed (eg. 'calc', 'var').
+	/**
+	 * The main font size UI displays a toggle group when the presets are less
+	 * than six and a select control when they are more.
+	 *
+	 * A select control is also used when the value of a preset cannot be
+	 * immediately computed (eg. 'calc', 'var') and there is no `alias` provided
+	 * for every font size.
+	 */
+	const fontSizesContainComplexValues = fontSizes.some(
+		( { size } ) => ! isSimpleCssValue( size )
+	);
+	const allFontSizesHaveAliases = fontSizes.every(
```

Comment:
Just a though — I wonder if the decision to display an alias should be made for each "font size" independently — i.e.:

- There are more than 6 presets? Display a select control
- Less than 6 presets:
  - Display a select control only if at least one font size is not a simple control, and doesn't have an alias either
  - Otherwise, display a toggle group

This mean that, as far as all "complex" values are paired with an alias, we should be able to show a toggle group

Reply:
Marco's suggestion here matches the preconceived notion I had of how this would work before I read the PR description.

My thinking was that the alias would act as an override or fallback. Something for theme authors to provide when we can't determine a sensible value from the font size. 

I can also see the other angle where this could lead to an inconsistent UI. A theme author adds aliases such as `XL` for complex values but omits aliases for simple ones, resulting in a mixture of numeric and string labels in the segmented control. A similar situation could arise if we wished to use the aliases for hints within the select control.

If we made the decision to display aliases at the individual font size level, it might alleviate the need to plumb through `shouldUseAliases`.

Reply:
>I wonder if the decision to display an alias should be made for each "font size" independently

I agree with Aaron:
>where this could lead to an inconsistent UI. A theme author adds aliases such as XL for complex values but omits aliases for simple ones, resulting in a mixture of numeric and string labels in the segmented control.